projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/datajobs/webhook/PostCreateWebHookProviderTest.java

```diff
@@ -6,11 +6,18 @@
 package com.vmware.taurus.datajobs.webhook;
 
 import lombok.Getter;
-import org.mockito.InjectMocks;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.TestPropertySource;
 
+@TestPropertySource(
+    properties = {
+      "datajobs.post.create.webhook.endpoint=http://localhost:4444",
+      "datajobs.post.create.webhook.internal.errors.retries=0",
+      "datajobs.post.create.webhook.authentication.enabled=false"
```

Comment:
Should we add integration test with authentication enabled?

Reply:
I don't know, a separate IT test for this particular feature is a bit of overhead.